macros/edr/dbt_artifacts/upload_dbt_exposures.sql

```diff
@@ -52,5 +54,8 @@
         'path': node_dict.get('path'),
         'generated_at': elementary.datetime_now_utc_as_string()
       }%}
+    {% set time_excluded_dict = flatten_exposure_metadata_dict.copy() %}
+    {% do time_excluded_dict.pop("generated_at") %}
+    {% do flatten_exposure_metadata_dict.update({"artifact_hash": local_md5(time_excluded_dict | string) if local_md5 else none}) %}
```

Comment:
We could also define the `artifact_hash` within the warehouse (`md5(...) as artifact_hash`) and then it'd work even if you're not on dbt-v1.4.0, but I don't see why it'd be worth the trouble.